trio_websocket/_impl.py

```diff
@@ -57,6 +58,10 @@
     :type use_ssl: bool or ssl.SSLContext
     :param subprotocols: An iterable of strings representing preferred
         subprotocols.
+    :param list[tuple[bytes,bytes]] extra_headers: A list of 2-tuples containing
```

Comment:
Is the type annotation over-specifying?  E.g. instead of list, tuple or any iterable is valid.

Reply:
I prefer a more relaxed type, but I had to match the wsproto type. We could ask wsproto to relax the type, or we could convert the caller's argument to `list`. I'm still pretty unfamiliar with type checking in Python, so I could go either way on this.

Reply:
Mostly was curious-- it's ok to leave as is, thank you.